### DISCUSSION TITLE
{"error":"unauthorized","reason":"You are not authorized to access this db."}

### DISCUSSION THREAD

**Discussion by saurabhprasadsah** [apache/couchdb#3859] [Q&A]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT give us access or passwords to your actual CouchDB! )

## Steps to Reproduce

[NOTE]: # ( Include commands to reproduce, if possible. curl is preferred. )

## Expected Behaviour

[NOTE]: # ( Tell us what you expected to happen. )

## Your Environment

[TIP]:  # ( Include as many relevant details about your environment as possible. )
[TIP]:  # ( You can paste the output of curl http://YOUR-COUCHDB:5984/ here. )

* CouchDB version used:
* Browser name and version:
* Operating system and version:

## Additional Context

[TIP]:  # ( Add any other context about the problem here. )

**Comment by saurabhprasadsah** [comment]
i{"error":"unauthorized","reason":"You are not authorized to access this db."}..i will be pass the same auth but not runnning in the database

**Comment by nickva** [comment]
Without more details and steps to reproduce it's hard to say what the issue is. `You are not authorized to access this db.` is a generic security ( authz ) error thrown.

Please review https://docs.couchdb.org/en/stable/intro/security.html it might help solve the issue.
